cartridges/int_alma/cartridge/scripts/helpers/almaCheckoutHelper.js

```diff
@@ -159,6 +165,44 @@ function isFragmentActivated() {
     return Site.getCurrent().getCustomPreferenceValue('ALMA_Fragment_Payment');
 }
 
+/**
+ * Return true if plan is activated
+ * @param {Object} paymentMethod payment method
+ * @param {Object} plan plan
+ * @returns {boolean} payment method ID
+ */
+function planIsActivated(paymentMethod, plan) {
+    var almaActivated = paymentMethod.getCustom().almaActivated.split(' | ');
```

Comment:
rely on user input isn't a good idea :)

maybe you could use [trim](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/trim)
and just expect `|`, like this
```
    var almaActivated = paymentMethod.getCustom().almaActivated.split('|');
```